TheMET/TheMET/ViewControllers/ArtistArtsViewController.swift

```diff
@@ -26,8 +26,15 @@ class ArtistArtsViewController: UIViewController, UITableViewDelegate, UITableVi
     
     private var loadingArtIds: [ArtID] = []
     
+    private var artistArtsList: [ArtCellData] = []
+    
+    private var isSearchFailed: Bool = false
```

Comment:
This property is not easy to keep in sync with `contentStatus`.
I recommend instead introduce a reason as an associated value to `LoadingStatus.failed`.  It can be an enum/bool value on your choice. `LoadingStatus` is currently used by other classes, which should use `Void` as a failed-reason associated type.
Alternatively, a new enum `Artists LoadingStatus` can be introduced to be used instead of generic `LoadingStatus`.